src/vcpkg/binarycaching.cpp

```diff
@@ -459,7 +454,7 @@ namespace
 
         RestoreResult try_restore(const InstallPlanAction&) const override { return RestoreResult::unavailable; }
 
-        void push_success(const InstallPlanAction&) const override { }
+        void push_success(const BinaryProviderPushRequest&, MessageSink&) override { }
 
         void prefetch(View<InstallPlanAction> actions, View<CacheStatus*> cache_status) const override
```

Comment:
I also thought about that and it would have really helped me in another PR, but since this is not necessary here I would do that in another PR to keep this PR small.